django_ckeditor_5/static/django_ckeditor_5/app.js

```diff
@@ -89,6 +89,9 @@ function createEditors(element = document.body) {
                 wordCountWrapper.innerHTML = '';
                 wordCountWrapper.appendChild(wordCountPlugin.wordCountContainer);
             }
+            if(editorEl.hasAttribute("disabled")){ 
+                editor.enableReadOnlyMode( 'docs-snippet' );
```

Comment:
Thanks for your work on the code. Is it intended to use the same feature identifier for all editors? 
 Could you explain why the same feature identifier is used for all editors?"

Reply:
@hvlads sorry. I don't understand

Reply:
1) Could you please clarify the purpose of adding this here and how it's meant to be used?

2) Regarding the `lockId: string | symbol` parameter:
This parameter serves as a unique identifier for setting the editor into a read-only state. It's essential to ensure that each `lock Id` is unique within your codebase. 


editor.enableReadOnlyMode( 'my-feature-id' );
editor.enableReadOnlyMode( 'my-other-feature-id' );
// ...
editor.disableReadOnlyMode( 'my-feature-id' );
editor.isReadOnly; // `true`.
editor.disableReadOnlyMode( 'my-other-feature-id' );
editor.isReadOnly; // `false`.

Parameters
lockId : string | symbol
A unique ID for setting the editor to the read-only state.

In the loop, you're adding "docs-snippet" to all editors with the "disabled" attribute.

Shouldn't we do this?
const lockId = `docs-snippet-${id}`;
editor.enableReadOnlyMode(lockId);